bash/0005-bash-4.3-msys2-fix-lineendings.patch

```diff
@@ -156,6 +173,63 @@ index 2001b4e..3ba2029 100644
  	    }
  	  else
  	    break;
+diff --git a/tests/crlf.right b/tests/crlf.right
+new file mode 100644
+index 0000000..d7fd195
+--- /dev/null
++++ b/tests/crlf.right
+@@ -0,0 +1,8 @@
++Line with LF
++Line with
 CR
++Line with
 CRLF
++Line with
++ LF
++Line with
 CR
++Line with

```

Comment:
Lines 183, 184, and 187 contain CR, and line 188 ends with CRLF.
`bash/.gitattributes` has been added to keep the CRs.